deploy/helm/sumologic/conf/traces/traces.otelcol.conf.yaml

```diff
@@ -0,0 +1,51 @@
+receivers:
+  opencensus: {}
+  jaeger:
+    protocols:
+      grpc:
+  zipkin: {}
+processors:
+  k8s_tagger:
+    passthrough: false
+    extract:
+      metadata:
+        # extract the following well-known metadata fields
+        - podName
+        - deployment
+        - cluster
+        - namespace
+        - node
+        - startTime
+  queued_retry:
+    num_workers: 16
+    queue_size: 10000
+    retry_on_failure: true
+  batch:
+    send_batch_size: 128
+    timeout: 5s
+extensions:
+  health_check: {}
+exporters:
+  # opencensus:
+  #   compression: "gzip"
+  #   endpoint: "someopencensus.target.com:443" # Replace with a real endpoint.
+  #   num_workers: 8
+  #   reconnection_delay: 2s
+  #   secure: true
+  # zipkin:
+  #   url: "http://somezipkin.target.com:9411/api/v2/spans" # Replace with a real endpoint.
+  # logging:
+  #   loglevel: debug
+  jaeger_grpc:
+    endpoint: "jaeger:14250" # Replace with a real endpoint.
```

Comment:
we want to use zipkin => `http://jaeger:9411/api/v2/spans`

Reply:
When the fluentd-zipkin will be merge, even `http://{{ sumologic.fullname }}:9411/api/v2/spans`

Reply:
Fixed.

Reply:
Note this means that both of those exporters will be enabled and trying to connect

Reply:
@pmm-sumo is it OK now?